pages/docs/extensions.mdx

```diff
@@ -1,22 +1,22 @@
+We are working on making it as complete as possible.
 
 You can then download and install extensions from the extension directory directly in your FOSSBilling installation.
 
-## Payment adapters
-
-| **Payment Adapter** |                | **Features** |        | **Developer**                                                | **Status**   |
-| ------------------- | -------------- | ------------ | ------ | ------------------------------------------------------------ | ------------ |
-|                     | Single Payment | Subscription | Refund |                                                              |              |
-| PayPal              | ✅              | ✅            | ❌      | FOSSBilling                                                  | Production\* |
-| Stripe              | ✅              | ❌            | ❌      | FOSSBilling                                                  | Production\* |
-| Razorpay            | ✅              | ❌            | ❌      | [Albinvar](https://github.com/albinvar/Razorpay-FOSSBilling) | Testing\*    |
-| Payrexx             | ✅              | ❌            | ❌      | Jaap Marcus                                                  | In Developent          |
-| Mollie              | ✅              | ❌            | ❌      | FOSSBilling                                                  | Production\*          |
-| UddoktaPay          | ✅              | ❌            | ❌      | [UddoktaPay](https://github.com/UddoktaPay/FOSSBilling)      | Production\* |
-| Bitcart             | ✅              | ❌            | ❌      | [Bitcart](https://github.com/bitcart/bitcart-fossbilling)    | Production\* |
-| Paystack           | ✅              | ❌            | ❌      | [samapraku](https://github.com/samapraku/boxbilling-paystack-adapter)    | Incompatible with FOSSBilling > 0.4 |
+## List of Payment adapters
+
+| **Payment Adapter** |                | **Features** | **Developer**                                                | **Status**   |
+| ------------------- | -------------- | ------------ | ------------------------------------------------------------ | ------------ |
+|                     | Single Payment | Subscriptions |                                                              |              |
+| PayPal              | ✅              | ✅         | FOSSBilling                                                  | Production\* |
+| Stripe              | ✅              | ❌         | FOSSBilling                                                  | Production\* |
+| Razorpay            | ✅              | ❌         | [Albinvar](https://github.com/albinvar/Razorpay-FOSSBilling) | Production\*    |
+| Payrexx             | ✅              | ❌         | Jaap Marcus                                                  | In Developent          |
+| Mollie              | ✅              | ❌         | FOSSBilling                                                  | Production\*          |
```

Comment:
Unless in main repo we should include a link to the repo...

Reply:
Added it.